src/content/docs/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -96,29 +101,31 @@ These components work together to create distinct advantages:
 
 We recommend that you explore both New Relic and OpenTelemetry options. With New Relic instrumentation, there are inherent advantages to developing instrumentation and platform features which work together, and New Relic integrations tend to work better out of the box. On the other side, OpenTelemetry offers an unparalleled degree of flexibility and control, but this sometimes requires additional research and effort to achieve the desired outcome.
 
 ## OpenTelemetry is continually evolving [#emerging-standard]
 
-The OpenTelemetry project has a large scope which has grown over the years. While many core components have reached stability (including OTLP, the trace API / SDK, the metric API / SDK, the log bridge API / SDK, http semantic conventions, and many language implementations), there are naturally pieces which are at various other stages of maturity.
+The OpenTelemetry project has a large scope which has grown over the years. While many core components have reached stability (including OTLP, the trace API/SDK, the metric API/SDK, the log bridge API/SDK, HTTP semantic conventions, and many language implementations), there are naturally pieces which are at various other stages of maturity.
 
-New Relic aims to have first class support for OpenTelemetry, including ingesting all OTLP data into our general purpose observability platform and building user experiences on top of OpenTelemetry data to help drive insights from the data out of the box. As components emerge and develop our platform capabilities will evolve alongside. However, please be cognizant of the maturity status of any OpenTelemetry components you are integrating with. While we try to stay on top of changes, it can be challenging to build around breaking changes of experimental components.
+New Relic aims to have first-class support for OpenTelemetry, including ingesting all OTLP data into our general purpose observability platform and building user experiences on top of OpenTelemetry data to help drive insights from the data out of the box. As components emerge and develop, our platform capabilities will evolve alongside. However, please be cognizant of the maturity status of any OpenTelemetry components you are integrating with. While we try to stay on top of changes, it can be challenging to build around breaking changes of experimental components.
```

Comment:
```suggestion
New Relic aims to have first-class support for OpenTelemetry, including ingesting all OTLP data into our general purpose observability platform and building user experiences on top of OpenTelemetry data to help drive insights from the data out of the box. As components emerge and develop, our platform capabilities will evolve alongside. However, please be cognizant of the maturity status of any OpenTelemetry components you're integrating with. While we try to stay on top of changes, it can be challenging to build around breaking changes of experimental components.
```